js/tabs.js

```diff
@@ -0,0 +1,56 @@
+var tabs = document.querySelectorAll('.js-tab');
+
+function setupTabs() {
+  for (var i = 0; i < tabs.length; i += 1) {
+    var tab = tabs[i],
+        tabSelector = tab.getAttribute('href'),
+        tabContent = document.querySelector(tabSelector);
```

Comment:
I don't think we have a hard rule on this but I think being explicit about variable declarations is easier to read, e.g.:
```
var tab = tabs[i];
var tabSelector = tab.getAttribute('href');
var tabContent = document.querySelector(tabSelector);
```